src/main/web_src/src/components/fagsystem/udistub/visning/partials/UtvistMedInnreiseForbud.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { TitleValue } from '~/components/ui/titleValue/TitleValue'
+import Formatters from '~/utils/DataFormatter'
+import _get from 'lodash/get'
+
+export const UtvistMedInnreiseForbud = ({ oppholdsstatus }: any) =>
```

Comment:
Kunne kanskje laget en felles type for oppholdsstatus her og i AvslagEllerBortfall.tsx? 🤔